bg/index.html

```diff
@@ -0,0 +1,86 @@
+</head>
+<body>
+  <main>
+    <h1>Не питай, за да питаш, просто питай!</h1>
+    <p>
+      Понякога в някоя случайна чат стая в интернет някой пита нещо от сорта на:
+    </p>
+    <blockquote>
+      <span class="name">Гошо123:</span>
+      <p class="message">
+        Има ли експерти в Java тук?
+      </p>
+    </blockquote>
+    <p>
+      Това е лошо изказване поради много причини. Нещото, което човека
+      <em>наистина</em> пита е:
+    </p>
+    <blockquote>
+      <span class="name">Гошо123:</span>
+      <p class="message">
+        Има ли експерти в Java тук които биха се ангажирали да погледнат моят
+        проблем, независимо какъв е той, дори и ако не е наистина свързан с Java
+        или ако някой който не знае нищо за Java има възможността да ми отговори?
+      </p>
+    </blockquote>
+    <p>
+      Има много причини затова защо хората които ИМАТ знанието не биха си признали.
+      Като питате, Вие питате за повече от това, което си мислите че питате.
+    </p>
+    <p>
+      Питате хората да вземат отговорност. Вие разпитвате увереността на хората в техните способности.
+      Вие също излишно заграждате някои хора. Аз също отговарям на въпроси свързани с езици или библиотеки,
+      които никога не съм използвал, защото въпросите са (в програмистки начин) здрав разум.
+    </p>
+    <p>
+      Алтернативно може да се интерпретира като...
+    </p>
+    <blockquote>
+      <span class="name">Гошо123:</span>
+      <p class="message">
+        Имам въпрос за Java но съм твърде мързелив за да го формализирам в думи,
+        освен ако има някой в този канал който може да отговори
+      </p>
+    </blockquote>
+    <p>
+      ..което е просто мързеливо. Ако не искате да свършите работата за да решите проблема Ви, защо 
+      трябва ние да го направим?
+    </p>
+    <p>
+      Решението е да не питате, за да питате, но просто да питате. Някой който седи на празен ход 
+      в канала и само понякога поглежда в това, което се случва в канала е малко вероятен да Ви отговори на
+      "питай за да питаш" въпроса, но истинското Ви описание може да вземе интереса им и да Ви помогне
+      да получите отговор.
+    </p>
+    <p>
+      Така, за да обобщим, не питайте
+      <em>"Има ли експерти в Java тук?"</em>,
+      а по-скоро питайте
+      <em>"Как да направя [проблем] с Java и [друга релевантна информация]"</em>
+    </p>
+    <p>
+      Други сходни проблеми: <a href="https://xyproblem.info/">The XY Problem</a>, <a href="https://nohello.net/">No Hello</a>.
+      Допълнителна информация: <a href="https://stackoverflow.com/help/how-to-ask">How do I ask a good question?</a>,
+      или ако имате повече време: <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a>.
+    </p>
+  </main>
+  <footer>
+    Съдържанието е главно откраднато от <a href="https://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -

```

Comment:
As a translator, you could add a link to your Github profile here. I would recommend doing that to stay consistent with other versions of the website

Reply:
Sure, just a sec